packages/devtools_app/lib/src/framework/release_notes/release-notes-next.md

```diff
@@ -0,0 +1,16 @@
+This is draft for future release notes, that are going to land on
+[the Flutter website](https://docs.flutter.dev/development/tools/devtools/release-notes).
+
+# DevTools 2.18.0 release notes
+
+Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
+
+## General Updates
+This release included several bug fixes and improvements, as well work
+towards some new features that are coming soon!
+See the full list of commits for DevTools 2.18.0
+[in the changelog](https://github.com/flutter/devtools/blob/master/CHANGELOG.md#2180).
+
```

Comment:
add a header for each page - see the template here: https://docs.google.com/document/d/16UGZX_X2GYn26xHeeFtHsvMdyJ96NCplBQxB0KhE-2g/edit